src/Modules/Family/RegistrationContainer.js

```diff
@@ -73,7 +73,7 @@ const RegistrationContainer = (props) => {
     }
     try {
       await axios.post(CREATE_RESERVATION, {
-        reservation: { event_date_id, event_slot_id }
+        reservation: eventSlotId === undefined ? {event_date_id} : {event_date_id, event_slot_id}
```

Comment:
```suggestion
        reservation: eventSlotId ? {event_date_id} : {event_date_id, event_slot_id}
```
`undefined` is already falsey.